redis.go

```diff
@@ -315,6 +315,9 @@ func (c *baseClient) initConn(ctx context.Context, cn *pool.Conn) error {
 	if !c.opt.DisableIndentity {
 		libName := ""
 		libVer := Version()
+		if c.opt.ClientNameSuffix != "" {
```

Comment:
Can you check the format vs nredisstack (ping @shacharPash ) or redis-py (@dvora-h )